optaplanner-core/src/main/java/org/optaplanner/core/config/partitionedsearch/PartitionedSearchPhaseConfig.java

```diff
@@ -71,11 +68,11 @@
     // Constructors and simple getters/setters
     // ************************************************************************
 
-    public Class<SolutionPartitioner> getSolutionPartitionerClass() {
+    public Class<? extends SolutionPartitioner<?>> getSolutionPartitionerClass() {
         return solutionPartitionerClass;
     }
 
-    public void setSolutionPartitionerClass(Class<SolutionPartitioner> solutionPartitionerClass) {
+    public void setSolutionPartitionerClass(Class<? extends SolutionPartitioner<?>> solutionPartitionerClass) {
```

Comment:
This is necessary in order to be able to use an implementation of the `SolutionPartitioner` interface.